src/Interactive/Features/Interactive/Core/InteractiveHost.Service.cs

```diff
@@ -740,6 +740,9 @@ private bool Execute(string text)
                 return true;
             }
 
+            private delegate ITypeSymbol GetSubmissionResultTypeDelegate(Compilation compilation, out bool hasValue);
+            private GetSubmissionResultTypeDelegate GetSubmissionResultType = typeof(Compilation).GetTypeInfo().GetDeclaredMethod("GetSubmissionResultType").CreateDelegate<GetSubmissionResultTypeDelegate>();
```

Comment:
Does this allow tests to pass? If not, consider creating a delegate that returns an error type instead.